packages/regl-worldview/src/commands/FilledPolygons.js

```diff
@@ -50,18 +50,17 @@ type Props = {
 export default function FilledPolygons({ children: polygons = [], getHitmapId }: Props) {
   const triangles = [];
   for (const poly of polygons) {
-    const { points } = poly;
+    const { points, id, color } = poly;
     const pose = poly.pose ? poly.pose : NO_POSE;
     const earcutPoints: Vec3[] = getEarcutPoints(points);
     const polyPoints: Point[] = earcutPoints.map(([x, y, z]) => ({ x, y, z }));
-    const color = poly.color;
     triangles.push({
+      id,
```

Comment:
oops good catch. we should just spread the rest of the props from the `poly` into this object.

```
const { points, color, ...restProps } = poly;
.
.
.
triangles.push({
  ...restProps,
  points: polyPoints,
})

```